source/backend/ltsa/LtsaService.cs

```diff
@@ -222,6 +227,11 @@ private async Task<TokenModel> RefreshAccessTokenAsync(TokenModel token)
                     _logger.LogError(ex, $"Failed to send/receive auth refresh request: ${this.Options.AuthUrl}");
                     throw new LtsaException(ex.Message, ex, ex.StatusCode.Value);
                 }
+                catch (JsonException ex)
+                {
+                    _logger.LogError(ex, $"Failed to parse refresh token JSON response");
+                    new LtsaException(ex.Message, HttpStatusCode.InternalServerError);
+                }
```

Comment:
See github's comment above - perhaps you forgot to rethrow the exception?

![image](https://user-images.githubusercontent.com/24322224/202573245-4e8c7f76-5c70-4ed8-91e7-bef94205b59d.png)